sdk/identity/azure-identity/src/test/java/com/azure/identity/util/TestUtils.java

```diff
@@ -74,6 +75,11 @@ public Date expiresOnDate() {
                 // Access token dials back 2 minutes
                 return Date.from(expiresOn.plusMinutes(2).toInstant());
             }
+
+            @Override
+            public AuthenticationResultMetadata metadata() {
```

Comment:
We shouldn't need to do this ideally.
Appears to be a breaking change on msal.

Reply:
reported it to them. Leaving this for now as it is just in test code anyway.